docs/dashboards/layout-description.md

```diff
@@ -1,20 +1,33 @@
 Layout
 ===
 
+## Overview
 An essential part of any dashboard is the positioning of its components. The layout can initially be defined from the configuration object, and then changed when Edit mode is enabled. Here are the building blocks of a dashboard layout.
 
 ![layout-description-img.png](layout-description-img.png)
 * Green boxes: rows
 * Red boxes: individual cells
 * Blue boxes: nested layout inside a cell
 
-To be able to use Dashboards with layout system and editmode you first have to load the `layout` module.
+To be able to use Dashboards with layout system and edit mode you first have to load the `layout` module.
 
 ```html
 <script src="https://code.highcharts.com/dashboards/dashboards.js"></script>
 <script src="https://code.highcharts.com/dashboards/modules/layout.js"></script>
 ```
 
+Alternatively, you can download the NPM package like:
+```bash
+npm install @highcharts/dashboards
+```
+and import it in your project like:
+```js
+import * as Dashboards from '@highcharts/dashboards';
+import LayoutModule from '@highcharts/dashboards/modules/layout';
+
+LayoutModule(Dashboards);
+```
+
 If you prefer to use your own layout structure, feel free to use it as well. Only what you need is disable the gui by option. Please remember that each container should have an unique `id` for rendered component.
```

Comment:
Is this 100% correct? It will suffice to just exclude the `gui` object from the config. You only have to explicitly disable it by using `gui.enabled: false` if `gui` is actually included.

Reply:
On second thought I guess it is correct, since we are talking about layout.js and how to disable it if included.